src/Chirp.Razor/Pages/Public.cshtml.cs

```diff
@@ -21,7 +21,7 @@ public ActionResult OnGet(int pageNumber = 1)
     {
         Cheeps = _service.GetCheeps(pageNumber);
         CheepsPerPage = _service.CheepsPerPage;
-        NumOfCheeps = Cheeps.Count();
+        NumOfCheeps = _service.GetCheepCount();
```

Comment:
I believe this was the error, because we went from getting the amount of cheeps from the database, where count was just getting the amount of cheeps we requested (which was always 5).